server/src/main/java/org/elasticsearch/action/search/MultiSearchRequest.java

```diff
@@ -186,6 +188,7 @@ public static void readMultiLineFormat(BytesReference data,
             // support first line with \n
             if (nextMarker == 0) {
                 from = nextMarker + 1;
+                deprecationLogger.deprecated("support for empty first line in msearch API is deprecated and will be removed");
```

Comment:
I wonder if this is not misleading. In this version the first line is just ignored while in master we'll consider it as an empty action so I think that the message here should be more explicit.

Reply:
I agree it is misleading. yet I believe it will be regardless of the message that we come up with, because it is a weird change. we want to drop supporting an empty first line in case it does not represent an empty action metadata line, while we want to be able to support empty lines for action metadata lines in the first row